pkgs/tools/networking/tcpdump/default.nix

```
@@ -31,8 +23,8 @@ stdenv.mkDerivation rec {
 
   meta = {
```

```suggestion
  meta = with lib; {
```

and drop the the `lib` qualifiers below.